widgetastic_manageiq.py

```diff
@@ -740,17 +741,17 @@ class Calendar(TextInput):
     Args:
         name: "name" property of the readonly calendar field.
     """
-
     def fill(self, value):
+        # input = self.browser.element(self.name)
         if isinstance(value, date):
             date_str = value.strftime('%m/%d/%Y')
         else:
             date_str = str(value)
         self.move_to()
         # need to write to a readonly field: resort to evil
         if self.browser.get_attribute("ng-model", self) is not None:
-            # self.set_angularjs_value(self, date_str)
-            raise NotImplementedError
+            self.browser.execute_script(js.set_angularjs_value_script, self.browser.element(self),
+             date_str)
```

Comment:
Looks good, you just need to copy the string with the script over to this module, probably to this class, because this module is going to get ripped out to a separate package and there is no js.py :)